.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  lint-report:
+    name: Lint report
+    runs-on: ubuntu-latest
```

Comment:
Don't we want to run tests on `20.04` instead of latest?

Reply:
Fixed